### DISCUSSION TITLE
Keyboard shortcuts list

### DISCUSSION THREAD

**Discussion by urbanmichal** [datalust/seq-tickets#1851] [Help]
Hi,

I'm new to Seq. I love it so far, BUT I couldn't find a keyboard shortcuts list. It'd be very valuable, as it will speed up Seq usage.

**Comment by liammclennan** [comment]
Hi @urbanmichal 

Seq does not have many keyboard shortcuts. 

`ctrl+/` will search signals, queries, history and variables. Other than that there is the normal web keyboard navigation by tabbing between controls. 

Which keyboard shortcuts do you think would help make your Seq usage more efficient?

**Reply by Ahmad-anility** [reply]
I think in the dashboard some shortcuts would be helpful like:

- *F* Find dashboard
- *R* refresh
- *T* Tail

Having a VScode like CTRL+SHIFT+P would be next level. If it shows a window to auto-complete options like
- From Now
- Last 5m
- Last 15m
- ....
- followed by list of dashboard
- Go to Event
- Go to alert
